cmd/observe/observe.go

```diff
@@ -219,8 +220,12 @@ programs attached to endpoints and devices. This includes:
 	)
 	observerCmd.Flags().StringVarP(
 		&output, "output", "o", "",
-		"Specify the output format, one of:\n -compact:\tcompact output\n -dict:\teach flow is shown as KEY:VALUE pair\n -json:\tJSON encoding\n -table:\ttab-aligned columns",
-	)
+		`Specify the output format, one of:
+ compact:  Compact output
+ dict:     Each flow is shown as KEY:VALUE pair
+ json:     Deprecated. Use '--output raw-json' instead.
```

Comment:
Why is `json` being deprecated?

Reply:
i can go either way, but i wanted to deprecate this for one specific reason. right now the Flow type has node_name and time fields. these fields are redundant because they are also included at the top level.

if we can deprecate this current json output format, it also makes it easier to deprecate node_name/time fields inside the Flow type.

another aspect is that i simply wanted to avoid having 2 similar json output formats.

for now i'll remove the deprecation warning. let's have a separate discussion.